index.bs

```diff
@@ -713,9 +705,9 @@ action that corresponds to the first matching condition from the following:
 
 </div>
 
-Issue: The above algorithm should probably only apply to within the AnimationWorkletGlobalScope?
-
-[Issue](https://github.com/w3c/web-animations/issues/191)
+Issue(w3c/csswg-drafts#2071): The above interface exposes a conservative
+of subset of GroupEffect proposed as part of web-animation-2. Once that is available we should
```

Comment:
Accidental 'of' before 'subset' (Think it should be "The above interface exposes a conservative subset of GroupEffect")